feature/google/src/test/java/BillingWrapperTestBase.kt

```diff
@@ -57,7 +64,14 @@ open class BillingWrapperTestBase {
 
     @Before
     fun setup() {
+        clearAllMocks()
+        storeProducts = null
+        purchasesUpdatedListener = null
+        billingClientStateListener = null
```

Comment:
We were not cleaning mocks for all these tests afaik, which is surprising... I'm not sure if mockk is clearing mocks automatically on its own on each test... But in any case, I thought clearing mocks manually was safer. I would actually prefer to recreate all the mocks on each test, but this should also work.